packages/kit/src/core/create_app/index.js

```diff
@@ -3,12 +3,16 @@ import path from 'path';
 import { s } from '../../utils/misc.js';
 import { mkdirp } from '../../utils/filesystem.js';
 
-/** @type {Map<string, string>} */
+/**
+ * Maps file contents to file path.
+ * @type {Map<string, string>}
+ */
 const previous_contents = new Map();
 
 /**
- * @param {string} file
- * @param {string} code
+ * Checks `previous_contents` map for `file` key and performs strict inequality comparison of `code` against the value, then writes `code` to `file` if there are changes.
```

Comment:
The description is currently a bit too long and isn't really much easier to read than the code :smile: How about?
```suggestion
 * Checks `previous_contents` map for `file` and writes `code` to `file` if there are changes.
```

Reply:
I'd go further — I'd suggest that the `previous_contents` stuff is all implementation detail, irrelevant to understanding what the function's purpose is. All you really need to know is that it writes a file if the contents have changed. But then again you can figure that out from the name of the function!